resources/html/configure.html

```diff
@@ -231,6 +231,19 @@ <h6>Enable Catalogs:</h6>
                             </label>
                         </div>
                     </div>
+
+                    <!-- Enable Proxy Debrid Stream Checkbox -->
+                    <div id="proxy_debrid_section" class="mb-3">
+                        <h6>Proxy debrid stream:</h6>
+                        <div class="form-check">
+                            <input class="form-check-input" type="checkbox" name="proxy_debrid_stream"
+                                   id="proxy_debrid_stream" {% if user_data.streaming_provider and user_data.proxy_debrid_stream %}checked{% endif %}>
+                            <label class="form-check-label" for="proxy_debrid_stream">
+                                Proxy the video via Mediafusion for debrid <span class="bi bi-question-circle" data-bs-toggle="tooltip" data-bs-placement="top"
```

Comment:
@ragmehos can we only show this option when user selecting the Streaming provider other than Direct Torrent. You can look into the JS code for Watch list catalog which is also only showing when selecting other than direct torrent. 

Also add the indicator saying only available in ElfHosted Premium & Self Hosting users.

Reply:
Yes, that is how it's setup already unless I am doing something wrong. I validated by selecting direct and this option doesnt show up and only shows for other providers. Will do the other fix mentioned.